pandas/io/sql.py

```diff
@@ -1304,10 +1300,7 @@ def to_sql(
 
             for col, my_type in dtype.items():
                 if not isinstance(to_instance(my_type), TypeEngine):
-                    raise ValueError(
-                        "The type of {column} is not a "
-                        "SQLAlchemy type ".format(column=col)
-                    )
+                    raise ValueError(f"The type of {col} is not a " "SQLAlchemy type ")
```

Comment:
looks like there is an extra space after "type", and black formatting split this into two strings that should be manually re-joined.

Reply:
i'll join it in my next commit, thanks for the feedback